smart-contracts/test/test_bridgeBankLock.js

```diff
@@ -335,6 +398,19 @@ describe("Test Bridge Bank", function () {
     });
 
     it("should not allow user to multi-lock when parameters are malformed, invalid sif addresses", async function () {
+      // Add the tokens into white list
+      await state.bridgeBank.connect(operator)
+      .updateEthWhiteList(state.token1.address, true)
+      .should.be.fulfilled;
+
+      await state.bridgeBank.connect(operator)
+      .updateEthWhiteList(state.token2.address, true)
```

Comment:
Please change this whole file to indent statements that extend from previous lines.